sql/src/main/java/io/crate/executor/transport/AlterTableOperation.java

```diff
@@ -320,26 +319,29 @@ public AlterTableOperation(ClusterService clusterService,
 
         List<ChainableAction<Long>> actions = new ArrayList<>(7);
 
-        if (completeTableIsClosed == false) {
+        if (sourceIndices.length > 0) {
+            if (completeTableIsClosed == false) {
+                actions.add(new ChainableAction<>(
+                    () -> updateOpenCloseOnPartitionTemplate(false, sourceTableIdent),
```

Comment:
these action must be done even on an empty partitioned table to prevent inserts(partition creation) while renaming